tidy3d/web/api/container.py

```diff
@@ -13,12 +15,14 @@
 from ..api import webapi as web
 from ..core.task_info import TaskInfo, RunInfo
 from ..core.constants import TaskId, TaskName
-from ...components.base import Tidy3dBaseModel
+from ...components.base import Tidy3dBaseModel, cached_property
 from ...components.types import annotate_type
 from ...log import log, get_logging_console
 
 from ...exceptions import DataError
 
+# Max # of workers for parallel upload / download: above 10, performance is same but with warnings
+DEFAULT_NUM_WORKERS = 10
```

Comment:
Is this dependent on the user connection or our servers? This is more out of curiosity; 10 seems like a reasonable default to me.

Reply:
I don't know actually. @lei-flex ?

Reply:
@lucas-flexcompute it might be our severs, because I tried several things to change the configuration to allow more workers but it still gave me a warning above 10.